lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -55,15 +52,22 @@ def call(env)
           request_span.set_error(e)
           raise e
         ensure
+          # the source of truth in Rack is the PATH_INFO key that holds the
+          # URL for the current request; some framework may override that
+          # value, especially during exception handling and because of that
+          # we prefer using the `REQUEST_URI` if this is available.
```

Comment:
:+1: good to go, and anyway, this is orthogonal to / independent from the short-term goal we have: have rack support to avoid loosing traces.